portfolio.css

```diff
@@ -0,0 +1,113 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+header {
+  width: 100%;
+}
+
+li {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+.page-container {
+  display: flex;
+  flex-direction: column;
+}
+
+.header-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  height: 6.5vh;
+}
+
+.Nav-1 a {
+  color: #28352f;
+  font-size: 5.5vw;
+  padding-left: 5vw;
+}
+
+.Nav-2 a {
+  padding-right: 5vw;
+}
+
+.section-title {
+  font-size: 6.8vw;
+  text-align: center;
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  width: 100%;
+  color: #172b4d;
+  line-height: 12vw;
+}
```

Comment:
- [ ] The font size is bigger

Reply:
How much bigger should the font size be.